src/kOS/Communication/ConnectivityManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace kOS.Communication
+{
+    public interface ConnectivityManager
+    {
+        double GetDelay(Vessel vessel1, Vessel vessel2);
+    }
+}
```

Comment:
For now.  We can pretty easily adapt the command line to use multiple processors in an effort to test the communication stuff.  Like I said, doesn't need to happen right away, just something to think about.  I think it's a good idea to consider whether or not something can reside in safe for all new additions.